.c8rc

```diff
@@ -0,0 +1,7 @@
+{
+  "all": true,
+  "src": ["client/src", "server/src"],
```

Comment:
I'm a little confused. Does this `src` setting really take effect? If so, sounds like "client/out/test/" doesn't need to be excluded as it won't be included? I looked at latest build log, tools/build.mjs is also in the report.